libsodium-sys/Cargo.toml

```diff
@@ -1,25 +1,25 @@
 [package]
 authors = ["dnaq"]
 build = "build.rs"
-description = "FFI binding to libsodium"
+description = "FFI binding to libsodium generated with bindgen"
 documentation = "https://sodiumoxide.github.io/sodiumoxide"
 keywords = ["libsodium", "NaCl", "crypto"]
 license = "MIT OR Apache-2.0"
 links = "sodium"
 name = "libsodium-sys"
 repository = "https://github.com/sodiumoxide/sodiumoxide.git"
 categories = ["cryptography", "api-bindings"]
-version = "0.0.16"
+version = "0.0.17"
```

Comment:
This bump should ideally be its own PR, just as to keep the merge conflicts out of the way

Reply:
I agree.